### ISSUE TITLE
inconsistency between calibration and prediction in number of trips (implications for costs table)

### ISSUE THREAD

**Issue by gavinfay** [acarrharris/Fluke-MSE-simulation#7]
For each catch draw, the number of trips (n_draws) is (at most per period per state) 1,000 in the calibration steps, but is 10,000 in the prediction. 
This means there are only 1,000 trips in the cost table, but 10,000 trips that need costs assigning to them when making predictions.
As such - these trips get assigned NAs when merged, and then these are changed to 0, but still included.

I notice that in the calibration version 2 files, the number was 10,000 and consistent between the calibration and prediction.
Given the number of directed trips in the 'directed trips' table, 1000 seems small. (though I think these are scaled anyway and the 1,000 is to get the distribution?)

@acarrharris & @gdepiper should I:
1. change the 1,000 to 10,000 in the calibration, OR
2. recycle the cost data from the 1,000 trips in the calibration when merging the costs data to the predictions (i.e. use each trip's cost data 10 times), OR
3. change the 10,000 to 1,000 in the prediction.

**Comment by gavinfay** [comment]
Getting it correct is the most important thing, but option 3. would mean for a faster prediction step run time (smaller object sizes to work with).

**Comment by acarrharris** [comment]
The ceiling value of n_draws should be consistent between the two files. I changed it to 1,000 at some point in the calibration to reduce computing time while testing the model. 

We could start with the ceiling of 1,000 (option 3), then at some point re-run with a larger number of draws and compare computing time and output. I'm not sure 1,000 vs. 10,000 (or somewhere in between) will make a big difference in terms of output. A while ago I tested between simulating the actual number of draws needed (no scaling), and a ceiling of maybe 10,000 and found almost no difference between the resulting outputs.
